### DISCUSSION TITLE
[Enhancement] 希望DateTimePicker组件可以支持把TimePicker替换为TimeSelect

### DISCUSSION THREAD

**Discussion by dream998** [element-plus/element-plus#17424] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `N/A`

## Description

在使用DateTimePicker组件时存在时间只能选自定义的几个固定值的需求，类似于TimeSelect组件的形式，希望DateTimePicker可以把TimePicker替换为TimeSelect，同时建议TimeSelect支持自定义选项

### Additional comments

_(empty)_

**Comment by nasirovx** [comment]
Yes, I support this enhancement. Allowing DateTimePicker to replace TimePicker with TimeSelect would be very useful, especially for scenarios where selecting from a predefined set of times is required. Additionally, enabling TimeSelect to support custom options would provide even greater flexibility.
